README.md

```diff
@@ -1,24 +1,92 @@
-# gender-bias-web
```

Comment:
You should not be changing the `README` as part of this PR. 

See https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request